src/frontend/index.tsx

```diff
@@ -32,11 +34,14 @@ import(`i18n-iso-countries/langs/nb.json`).then(result => registerLocale(result)
 
 ReactDOM.render(
     <React.StrictMode>
-        <SprakProvider tekster={{ nb: norskeTekster }} defaultLocale={LocaleType.nb}>
-            <HttpProvider>
-                <App />
-            </HttpProvider>
-        </SprakProvider>
+        <ThemeProvider theme={{}}>
```

Comment:
Ble veldig usikker på om man faktisk trenger en Provider her nå siden vi ikke sender inn/har bruk for tema enda 🧐  har du prøvd uten? og i såfall, funker det? Hvis det funker uten kan vi kanskje droppe ThemeProvider til vi får bruk for det?

Reply:
Fjerner den :)